### ISSUE TITLE
Can't use MCU daughterboard with MOLEX auxiliary power connector.

### ISSUE THREAD

**Issue by JohnDCartwright** [dekuNukem/ATX4VC#2]
I was disappointed to find that on the 2 ATX4VCs that I have purchased the MCU daughter board overlaps the MOLEX auxiliary power connector, so both can not be used at the same time.
I had to mount the MCU daughterboards away from the ATX4VCs using a six pin extender cable.
Looking at the photo in the user manual, the MCU daughterboard does not overlap the auxiliary connector, so something has changed on the latest production!

![20240316_113403 1](https://github.com/dekuNukem/ATX4VC/assets/46418807/6ba21b24-7267-4467-8acc-8ac1a7188ad2)
![20240316_113412 1](https://github.com/dekuNukem/ATX4VC/assets/46418807/85947cf3-f8a9-4534-9f6f-f3a5311e57ff)

**Comment by dekuNukem** [comment]
Yeah the new revision of ATX4VC added POWER_GOOD signal to the terminal block, but I didnt notice that the daughterboard is now blocking the header!

I'm planning to design a new matching daughterboard as well, but in the meantime I'm happy to give you a refund for the daughterboard. Feel free to let me know the tindie order number.

**Comment by JohnDCartwright** [comment]
My Tinder order number is Order #460717, but refund not necessary thanks, I have got round the problem by mounting the daughterboard separately. Just wanted to let you know about the problem. 
Would be interested to learn about the new daughterboard when it arrives. 🙂
